packages/spotlight/SpotlightContainerDecorator/useSpotlightContainer.js

```diff
@@ -0,0 +1,100 @@
+import useClass from '@enact/core/useClass';
+import React from 'react';
+
+import SpotlightContainer from './SpotlightContainer';
+
+/**
+ * Configuration for `useSpotlightContainer`
+ *
+ * @typedef {Object} useSpotlightContainerConfig
+ * @memberof spotlight/SpotlightContainerDecorator
+ * @property {Object}   [containerConfig]                   Spotlight container configuration.
+ * @property {Function} [navigableFilter]                   Called to determine if a spottable
+ *                                                          component is accessible.
+ * @property {Boolean}  [preserveId]                        Preserves the spotlight id when the
```

Comment:
is it important to call out which properties can be updated and which are only set once at creation?